packages/eslint-plugin-next/lib/index.js

```diff
@@ -36,7 +35,6 @@ module.exports = {
         '@next/next/no-title-in-document-head': 1,
         '@next/next/google-font-display': 1,
         '@next/next/google-font-preconnect': 1,
-        '@next/next/link-passhref': 1,
```

Comment:
Should we only disable this when `legacyBehavior` isn't used and the `experimental.newNextLinkBehavior` config is enabled?

Reply:
The problem with this is that we don't have access to the next.config.js when running eslint yourself, which is why removing it is better for now 😢